### ISSUE TITLE
README suggests classes can be constructed using `new` but this does not appear to be the case.

### ISSUE THREAD

**Issue by yyny** [Download/mics#46]
As shown and stated [here](https://github.com/Download/mics/blob/develop/README.md#creating-instances-of-mixins).

Relevant example shown in README:
```js
var looker = new Looker()     // > A looker is born!
looker.look()                 // > Looking good!
looker instanceof Looker      // true
```

Specifically, [line 45](https://github.com/Download/mics/blob/develop/src/index.js#L45) of `index.js`  contains code which eventually returns an instance of `(...args) => new Class(...args)` (which has been like this for [over 6 months ago](https://github.com/Download/mics/blame/96dd5ec65bc57086924a02bd96eb9c8be3941eaa/src/index.js#L18)), which cannot be called as a constructor (Since it is an arrow function).

```js
new (() => {}); //=> Uncaught TypeError: (intermediate value) is not a constructor
                //=>    at <anonymous>:1:1
```

Using `function(...args) { return new Class(...args) }` instead would make the code work (without compilation) for platforms that disallow arrow functions as constructors (As required by ES6).

**Comment by mindfullsilence** [comment]
Yeah this is a bummer. I really like this library, powerful tool in my toolbelt. But the opinionated decision of trying to work around using`new` gets me down. Honestly I prefer `new`. It's easier to read when something is being instantiated in the middle of a few hundred lines of code.

**Comment by Download** [comment]
Hi guys, I see what the problem is and will look at this soon.
Sorry for the late response but this is Open Source and I do it in my spare time.
